drivers/integration/docker/docker.go

```diff
@@ -25,8 +25,9 @@ type driver struct {
 }
 
 type volumeMapping struct {
-	Name             string `json:"Name"`
-	VolumeMountPoint string `json:"Mountpoint"`
+	Name             string                 `json:"Name"`
+	VolumeMountPoint string                 `json:"Mountpoint"`
+	VolumeStatus     map[string]interface{} `json:"Status"`
```

Comment:
See the Docker spec.

https://github.com/docker/docker/pull/21006#issuecomment-210188532